20180619/README.md

```diff
@@ -0,0 +1,359 @@
+  save,
+  load,
+  remove
+};
+```
+
+Save this file as a `storage.js`.
+You can find these snippets [here](https://github.com/matchmore/js-sdk/tree/master/platform).
+
+This will be used later in while wiring the manager.
+
+## Setup the manager
+
+For simplicity, we will do everything in the App.js file, so lets setup _Matchmore_!
+
+In your App.js add these imports
+
+```javascript
+import { Manager, LocalStoragePersistenceManager } from "@matchmore/matchmore";
+import "./storage";
+```
+
+Put somewhere you Api key handy like
+
+```javascript
+const apiKey = "YOUR_API_KEY";
+```
+
+Now on the start of the application we should initialize the manager and wire in the persistence.
+
+In our App component let do
+
+```javascript
+async componentDidMount() {
+    PlatformConfig.storage = Storage;
+    const localPersistenceManager = new LocalStoragePersistenceManager();
+    await localPersistenceManager.load();
+    this.manager = new Manager(
+      apiKey, undefined, localPersistenceManager
+    );
+  }
+```
+
+And we need to setup our device as the _main device_, first created device is our main.
+
+```javascript
+let myDevice = await this.manager.createMobileDevice(
+    "me", //your device name, specific for the application
+    "ios", //platform
+    "DEVICE_TOKEN"); //token to route matches, like apns, fcm, websockets
+```
+
+## Make it work!
+
+Great! Now we are up and running!
+Now, we would want to know who is nearby, to do that, lets create a subscription.
+
+```javascript
+let datingSubscription = await this.manager.createSubscription(
+      "Dating App", //topic specific to your applications, you can have multiple topic per app, it is up to you how will you model it, we will provide some insights about this soon
+      10000, //range of subscriptions in meters, lets start of with 10km
+      86400, //duration of how long the subscription will last, in seconds, lets put in 24 hours
+      "age >= 18" // matching selector
+    );
+```
+
+The matching selector is a selector which will filter out only publications which you are interested in. Think of it as SQL for publications you will receive! Here we are saying we are looking only for publications with a property `age` greater or equal.
+
+Cool! Now we are waiting for that one! But we are missing one thing... We need to go out too! We need to _Publish_ ourself! Lets create a Publication!
+
+```javascript
+let datingPublication = await this.manager.createPublication(
+      "Dating App",
+      10000, //m
+      86400, //s
+      {age: 29, name: "Lucas"} //publication properties
+);
+```
+
+As we mentioned before, publication properties are the data we want to be searched by, our interests, picture url, age... whatever makes sense for your app.
+
+Ok, lets be real - we are just starting, nobody is our there, we need to create some "people" to test our and develop against.
+
+We can create virtual pins, which here we will mimic other people looking for their second half.
```

Comment:
`which here we will mimic other people` => `we will mimic with them other people` ?